src/client/styles/PasswordValidationStyles.ts

```diff
@@ -0,0 +1,73 @@
+import { css } from '@emotion/core';
+import validationGreenTick from '@/client/assets/svgs/validation-green-tick.svg';
+import validationRedCross from '@/client/assets/svgs/validation-red-cross.svg';
+import validationCross from '@/client/assets/svgs/validation-cross.svg';
+import eyeOpen from '@/client/assets/svgs/eye-open.svg';
+import eyeCrossed from '@/client/assets/svgs/eye-crossed.svg';
+
+export type ValidationStyling = 'success' | 'failure' | 'error';
+
+export const passwordValidatorsCss = css`
+  margin-bottom: 10px;
+`;
+
+export const validationInfoCss = (styling: ValidationStyling) => {
+  let color = '#121212';
+  if (styling === 'success') {
+    color = '#22874d';
+  } else if (styling === 'error') {
+    color = '#c70000';
+  }
+
+  return css`
+    font-family: GuardianTextSans, sans-serif;
```

Comment:
with font stuff, I suggest using the design system typography helpers: http://theguardian.design/2a1e5182b/p/95d5d0-code

this lets you customise the font through a method call rather than having to worry about family, size, style, weight, line-height etc.

think we have a few example of this being used in the project